### PR TITLE
chore: fix get RPO sample

### PR DESCRIPTION
**Pull request by tritone**
There was an extra line here that doesn't make sense. Fixing based on external user feedback.

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-storage/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes b/318488404 🦕

### CODE REVIEW

File: samples/snippets/storage_get_rpo.py [hunk 1]
```diff
@@ -35,8 +35,6 @@ def get_rpo(bucket_name):
 
     storage_client = storage.Client()
     bucket = storage_client.bucket(bucket_name)
```
Comment (cojenco) [inline samples/snippets/storage_get_rpo.py:37]:
```suggestion
    bucket = storage_client.get_bucket(bucket_name)
```

**Review by cojenco — COMMENTED** [review]
Thanks @tritone  for the fix! We'll also need to update to calling `get_bucket()` to actually make a metadata GET.

File: samples/snippets/storage_get_rpo.py (hunk 1 above)
Comment (tritone) [reply samples/snippets/storage_get_rpo.py]: changed; thanks!

**Review by cojenco — APPROVED** [review]
Re-running kokoro as the auth library fix just released, thank you!

**Pull request merged**